apps/admin-x-activitypub/src/views/Profile/Profile.tsx

```
@@ -234,230 +40,105 @@ const LikesTab: React.FC = () => {
+            isFetchingNextPage={isFetchingNextPage}
+            isLoading={isLoading}
+            noResultsMessage={`${handle || 'You'} have no followers yet`}
+        />
     );
 };
 
-type ProfileTab = 'posts' | 'likes' | 'following' | 'followers';
-
-interface ProfileProps {}
-
 const Profile: React.FC<ProfileProps> = ({}) => {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)